### PR TITLE
Feature/marshall/create user model

### PR DESCRIPTION
**Pull request by uimarshall**
Check the user model for review and merging.

### CODE REVIEW

**Review by cimthog — DISMISSED** [review]

File: jest-mongodb-config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    mongodbMemoryServerOptions: {
+        instance: {
+            dbName: 'jest',
```
Comment (enyeInc) [inline jest-mongodb-config.js:4]: I am curious, why is did you choose this db name?

File: models/User.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		type: String
+	},
+	date: {
+		type: Date,
+		default: Date.now
+	}
+});
+
+const User = mongoose.model("users", UserSchema);
+module.exports = User;
```
Comment (enyeInc) [inline models/User.js]: you can make this a one line
```
module.exports = mongoose.model("users", UserSchema);
```

File: jest-mongodb-config.js (hunk 1 above)
Comment (uimarshall) [reply jest-mongodb-config.js:4]: I got this from the docs on how to set up jest.
I guess i did not even understand what i was doing, so what am i suppose to use as dbName?

**Comment by enyeInc** [comment]
@uimarshall Can you merge master into this branch in order to remove the hound comments? I have already told you this several times now, this takes 1 min to do

**Comment by uimarshall** [comment]
I am surprised the hound messages is still showing because I have merge the master to it locally as directed.

**Pull request closed without merging**